ffac-update-location-gps/luasrc/usr/bin/update-location-gps

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/lua
+
+if not arg[1] then
+    print("Error: No /dev/tty* path specified. Please provide a TTY device path as the first argument.")
+    os.exit(1)
+end
+
+-- Get GPS device tty
+local TTY_DEVICE = arg[1]
+
+local uci = require('simple-uci').cursor()
+
+-- Use GPS as Stream
+local file = io.open(TTY_DEVICE, "r")
+
+while true do
```

Comment:
Can we somehow replace this while true with something else?
Maybe add a sleep to avoid this loop going crazy? In particular when a non-GPS tty device was plugged in.

Reply:
We could stop looking if the GPRMC line did not occur in the first 50 lines. This would be enough in any case.

Reply:
I only check the first 50 lines now - this still halts if no line is ever read at all from the stream.